web/less/room.less

```diff
@@ -380,6 +386,9 @@ body {
 
   .tc-dialog {
     max-width: 710px;
+    margin-left: 10px!important;
+    margin-right: 10px!important;
+    padding: 20px!important;
```

Comment:
These changes are not affected by large displays this will only apply and makes sure that margins on both left and right side are maintained at a minimum.

Actually added this to all dialogs makes all modal look better in the mobile view.

Please verify once